task/git-clone/0.1/git-clone.yaml

```diff
@@ -165,6 +172,18 @@ spec:
         fi
       }
 
+      check_symlinks() {
+        find $CHECKOUT_DIR -type l -print | \
```

Comment:
Parent image does not contain the find command .. so this command fails
`+ find /workspace/output/ -type l -print
/tekton/scripts/script-0-7m9xq: line 39: find: command not found`

Reply:
now the check is in different step that uses different image that has the find command available.